zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/domain/ObjectManagement.kt

```diff
@@ -32,13 +32,13 @@ data class ObjectManagement(
     @Column(name = "title", nullable = false, unique = true)
     val title: String,
 
-    @Column(name = "objecten_api_plugin_configuration_id", nullable = false)
+    @Column(name = "objecttypen_api_plugin_configuration_id", nullable = false)
```

Comment:
Isn't this a breaking change?

Reply:
We discussed this the Friday before my holiday, and I had the same concerns. @ThomasRitense mentioned that this feature was not used anywhere yet, so I could change it regardless.

Reply:
Just to be safe you could include a liquibase changeset to rename these columns

Reply:
I still agree with that, this feature is not in use yet. Either way is fine with me.

Reply:
I included a liquibase changeset